.buildkite/scripts/steps/integration_tests.sh

```diff
@@ -8,7 +8,7 @@ DEV=true EXTERNAL=true SNAPSHOT=true PLATFORMS=linux/amd64,linux/arm64 PACKAGES=
 
 # Run integration tests
 set +e
-TEST_INTEG_CLEAN_ON_EXIT=true SNAPSHOT=true mage integration:test
+TEST_INTEG_AUTH_ESS_REGION=azure-eastus2 TEST_INTEG_CLEAN_ON_EXIT=true SNAPSHOT=true mage integration:test
```

Comment:
Could we add an issue to revert this change and maybe add that issue to the next sprint or something, just so we don't forget about it?  Otherwise we'll keep going across data centers between the OGC-provisioned test runner hosts in GCP and ESS deployments in Azure.

And then maybe add a comment here in the code linking to that issue, again, to serve as a reminder?

Reply:
https://github.com/elastic/elastic-agent/issues/3444